app/helpers/dropdown_helper.rb

```diff
@@ -15,6 +15,30 @@ def organisation_options_for_select(selected = nil)
     options_for_select(organisation_options, selected)
   end
 
+  def topic_options_for_select(selected = nil)
+    topic_option_attributes = ->(topic) do
+      [
+        *topic.links.map { |link| link.target.title.squish },
+        topic.title.squish,
+        topic.content_id,
+      ]
+    end
+
+    topic_options = Content::Item
+                      .includes(links: %i(target))
+                      .where(
+                        document_type: 'topic',
+                        links: { link_type: Content::Link::PARENT },
+                      )
+                      .map(&topic_option_attributes)
+                      .sort_by { |parent_title, title, _| [parent_title, title] }
+                      .map { |parent_title, title, content_id|
```

Comment:
I think we should be using `do..end` for multi-line blocks; I'm surprised our linter didn't shout at you for this?

Reply:
Addressed in 5e545ac
cc/ @andrewgarner